SoftLayer/managers/network.py

```diff
@@ -774,3 +774,10 @@ def cancel_item(self, identifier, cancel_immediately,
                                 reason_cancel,
                                 customer_note,
                                 id=identifier)
+
+    def get_router(self):
+        """return routers account.
+
+        Returns routers.
+        """
+        return self.client.call('SoftLayer_Network_Pod', 'getAllObjects')
```

Comment:
Change the name to `get_pods` to be more descriptive and plural, add a datacenter parameter so we can get pods from a specific DC only to make the API request a little more efficient. I don't think you can filter by pod name sadly, but you can filter by datacenter which should be good enough.
 Make sure you call with a specific datacenter from the CLI command as well.

```suggestion
    def get_pods(self, datacenter=None):
        """Calls SoftLayer_Network_Pod::getAllObjects()

        returns list of all network pods and their routers.
        """
        _filter = None
        if datacenter:
            _filter = {"datacenterName":{"operation": datacenter}}'
        return self.client.call('SoftLayer_Network_Pod', 'getAllObjects', filter=_filter)
```